core/src/test/java/com/google/errorprone/bugpatterns/MemoizeConstantVisitorStateLookupsTest.java

```diff
@@ -162,4 +162,31 @@ public void negative_doesntMemoizeTwice() {
         .expectUnchanged()
         .doTest();
   }
+
+  @Test
+  public void testSuppressWarnings() {
```

Comment:
```suggestion
  public void suppressWarnings() {
```
It looks like we should drop the `test` prefix here, based on https://github.com/google/error-prone/commit/dbd04752b51372998f7fbd75d1e88ef731cefffe.

(As pointed out in https://github.com/google/error-prone/pull/3677#issuecomment-1370677956, we have a check for this in Error Prone Support which we are open to contributing upstream 😄, see related [documentation](https://error-prone.picnic.tech/bugpatterns/JUnitMethodDeclaration/))

Reply:
@rickie thanks!  I tried to resolve this, but the Google system to pull in the PR may have already kicked in...we'll see what actually lands :-)